### PR TITLE
Add more Faker classes

### PR DESCRIPTION
**Pull request by rolyatwilson**
This adds new classes and tests for:
- DragonBall
- FunnyName
- HitchhikersGuideToTheGalaxy
- Hobbit
- HowIMetYourMother
- LeagueOfLegends
- Overwatch
- Robin
- StarTrek

### CODE REVIEW

File: src/test/java/com/github/javafaker/DragonBallTest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.github.javafaker;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+
+public class DragonBallTest extends AbstractFakerTest {
```
Comment (codingricky) [inline src/test/java/com/github/javafaker/DragonBallTest.java]: Any chance you could use hamcrest matches and also put in assertions that do more than just asserting on whether something is empty or not, like maybe a regex. You can see how the other tests are written.

**Comment by denniss17** [comment]
Where are the actual values for these fakers? I would expect a change to `en.yml`

**Comment by codingricky** [comment]
@denniss17 I believe they’re already in the en.yml file. That file is a copy of the ruby one so it will have more data than we have classes for.

**Comment by codingricky** [comment]
@rolyatwilson can you make sure the FakerIT covers these classes too? Thanks

**Comment by rolyatwilson** [comment]
I’ll look into it.

**Comment by rolyatwilson** [comment]
@codingricky Thanks for mentioning `FakerIT`! I updated that to test the newly added classes. (apologies for the delay)

**Comment by rolyatwilson** [comment]
@codingricky is there anything else I need to do to get this PR ready to merge?

**Pull request merged**